patch-package
--- a/node_modules/rnkit-actionsheet-picker/android/app/build.gradle
+++ b/node_modules/rnkit-actionsheet-picker/android/app/build.gradle
@@ -1,8 +1,8 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 26
+    buildToolsVersion "26.0.3"
     defaultConfig {
         minSdkVersion 16
         targetSdkVersion 23
--- a/node_modules/rnkit-actionsheet-picker/android/app/src/main/AndroidManifest.xml
+++ b/node_modules/rnkit-actionsheet-picker/android/app/src/main/AndroidManifest.xml
@@ -2,7 +2,6 @@
     package="io.rnkit.actionsheetpicker">
 
     <application
-        android:allowBackup="true"
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
